exporter/awsemfexporter/metric_translator.go

```diff
@@ -75,9 +75,50 @@ type CWMetricStats struct {
 	Sum   float64
 }
 
+// Wrapper interface for:
+// 	- pdata.IntDataPointSlice
+// 	- pdata.DoubleDataPointSlice
+// 	- pdata.IntHistogramDataPointSlice
+// 	- pdata.DoubleHistogramDataPointSlice
+type DataPoints interface {
+	Len() int
+	At(int) DataPoint
+}
+
+// Wrapper interface for:
+// 	- pdata.IntDataPoint
+// 	- pdata.DoubleDataPoint
+// 	- pdata.IntHistogramDataPoint
+// 	- pdata.DoubleHistogramDataPoint
+type DataPoint interface {
+	IsNil() bool
+	LabelsMap() pdata.StringMap
+}
+
+// Define wrapper interfaces such that At(i) returns a `DataPoint`
+type IntDataPointSlice struct {
+	pdata.IntDataPointSlice
+}
+type DoubleDataPointSlice struct {
+	pdata.DoubleDataPointSlice
+}
+type DoubleHistogramDataPointSlice struct {
+	pdata.DoubleHistogramDataPointSlice
+}
```

Comment:
missed `IntHistogramDataPointSlice`?

Reply:
oh. I see. we're not using IntHistogramDataPointSlice currently.

Reply:
yup!